test/utils/describeConformanceV5.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-env mocha */
+import { expect } from 'chai';
+import * as React from 'react';
+import {
+  testClassName,
+  testPropsSpread,
+  describeRef,
+  testRootClass,
+  findRootComponent,
+  testReactTestRenderer,
+} from './describeConformance';
+
+/**
+ * Material-UI components have a `components` prop that allows rendering a different
+ * Components from @inheritComponent
+ * @param {React.ReactElement} element
+ * @param {() => ConformanceOptions} getOptions
+ */
+function testComponentsProp(element, getOptions) {
+  describe('prop: components', () => {
+    it('can render another root component with the `components` prop', () => {
+      const {
+        classes,
+        mount,
+        testComponentsRootPropWith: component = 'em',
+        componentsProp,
+      } = getOptions();
+
+      const wrapper = mount(React.cloneElement(element, { components: { Root: component } }));
```

Comment:
Would it make sense to use react-testing-library for this new test instead of enzyme?

Reply:
A bit hard to change it only for this test, as the other suits are coming from the original `describeConformance`. I would suggest, we do it in one run after all components are migrated and we can safely remove the old test suite.